apps/server/src/controllers/start-race.controller.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  AppPlayerId,
+  AppTournamentId,
+  socketProtocols,
+  StartRaceAccept,
+} from '@razor/models';
+import { store } from '@razor/store';
+
+import { PubSubEvents } from '../models';
+import { ContextOutput, Logger, pubsub } from '../services';
+import { generateRaceText } from '../utils';
+
+interface StartRaceRequestArgs {
+  context: ContextOutput;
+  playerId: AppPlayerId;
+  tournamentId: AppTournamentId;
+}
+
+const logger = new Logger('start-race.controller');
+
+export const startRaceController = async ({
+  context,
+  playerId,
+  tournamentId,
+}: StartRaceRequestArgs): Promise<void> => {
```

Comment:
As we have already discussed shall we send  error messages back to the user

Reply:
Opened a separate issue for this https://github.com/the-ai-team/razor/issues/80